e3sm_to_cmip/mpas.py

```diff
@@ -54,8 +54,7 @@ def remap(ds, mappingFileName, threshold=0.05):
         raise subprocess.CalledProcessError(
             'ncremap returned {}'.format(proc.returncode))
 
-    ds = xarray.open_dataset(outFileName, decode_times=False,
-                             mask_and_scale=False)
```

Comment:
I can't recall why I set ` mask_and_scale=False` in the first place and it's just part of a big initial commit of MPAS handlers so nothing helpful there.  I can't think of any advantage, since NaNs get replaced with a proper fill value on output either through CMOR or through the `write_netcdf` function.